.gitignore

```diff
@@ -3,6 +3,7 @@
 *.sig.key
 *.sym.key
 .cargo/advisory-db/
+.cargo/advisory-db..lock
```

Comment:
was this intentional?

Reply:
It was.  That file was always empty when it was created and while I'm generally in favor of committing lock files I wasn't finding anything useful regarding that particular file. It also just felt weird with the .. in the filename so I decided to add it to the .gitignore now and reverse that decision later if appropriate.